torch/_ops.py

```diff
@@ -46,6 +46,12 @@ def __call__(self, *args, **kwargs):
     def __getattr__(self, key):
         return getattr(self._op, key)
 
+    def __eq__(self, other):
+        if self._op == other._op:
+            return True
+        else:
+            return False
```

Comment:
You don't need an if statement here. Just return `self._op == other._op`. But it would be even better if OpOverload objects were singletons; then you could use ordinary object identity for the equality test here. (But see also BC considerations when there is a type mismatch).